src/js/gi/components/profile/Outcomes.jsx

```diff
@@ -36,6 +36,15 @@ class Outcomes extends React.Component {
         </div>
 
         <div className="medium-12 column">
+          <svg className="graph-key" viewBox="0 0 650 50">
```

Comment:
Can we potentially extract this SVG into a component instead of having it inline like this?